.gitignore

```diff
@@ -0,0 +1 @@
+protoc
```

Comment:
This file needs a newline at the end of the file so that it works across editors.

Reply:
Newline at end of file will be applied across all files in the next PR.